doc/config.md

```diff
@@ -168,6 +170,16 @@ Default:
 
 You can check the [parameter choice for pbkdf2](https://cryptosense.com/parameter-choice-for-pbkdf2/) for more information.
 
+## `Pubsub`
+
+Options for configuring the pubsub subsystem.
+
+### `Router`
```

Comment:
Can we, not support this? It would mean we have to always bundle two implementations in the browser when 99.9% of people are very likely to use only gossipsub.

Reply:
As discussed on IRC we're going to go with support this in Node.js and just throw "Router unavailable. Configure libp2p.modules.pubsub to use the ${config.Pubsub.Router} router." in the browser.